doctr/datasets/cord.py

```diff
@@ -60,7 +60,7 @@ def __init__(
 
         # List images
         tmp_root = os.path.join(self.root, 'image')
-        self.data: List[Tuple[Union[str, np.ndarray], Dict[str, Any]]] = []
+        self.data: List[Tuple[Union[str, np.ndarray], Union[Dict[str, Any], str]]] = []
```

Comment:
I think we need to return a dict in fact that we support datasets with more than one annotation (boxes/labels) something like
{image: ..,
boxes: .. (can be None),
labels: .. (can be None),
} 
to hold some key informations wdyt ?

Reply:
Mmmmh, about getitem:
- dict have a `.get` method, I'd argue we should limit the keys to the target task ("labels" for text reco)

About self.data:
- the simplest / lightest data format that fit the target task (string for text recognition)

So the getitem method need to pack this into a dict

Reply:
Sounds good

Reply:
@frgfm after thinking again about the output i would say it is better to return only a dict if we have more then 2 annotations (labels and boxes) otherwise we return the plain where the output should be clear. 
Only recognition has str annotations and pure detection has box annotations.
OCR-end-to-end is the only format with dict where we need both. wdyt ?

Reply:
I'd say it's dangerous to have heterogenous target data types (dict vs non-dict)

That makes transforms are to implement :sweat_smile:

Reply:
Mh yes i agree will update this next week that all annotations are provided in a dict (convert in __getitem__)